app/controllers/home_controller.rb

```diff
@@ -1,4 +1,21 @@
 class HomeController < ApplicationController
   def index
+    if user_signed_in?
+      @post=Post.new
+      @posts=Post.all
+=begin
```

Comment:
I think your feed should display post `recent first`

Reply:
Done